README.md

```diff
@@ -55,7 +55,7 @@ A curated list of awesome research code, software, manuals, and more on Git, dev
 
 - [asreview](https://github.com/asreview/asreview) - Active learning for systematic reviews
 - [bain](https://github.com/cjvanlissa/bain) - Bayes Factors for Informative Hypotheses
-- [mice](https://github.com/amices/mice) - Multivariate Imputation by Chained Equations
+- [mice](https://github.com/amices/mice) - Multivariate Imputation by Chained Equations (+ [ggmice](https://github.com/amices/ggmice) for visualizations)
```

Comment:
```suggestion
- [mice](https://github.com/amices/mice) - Multivariate Imputation by Chained Equations 
   * [ggmice](https://github.com/amices/ggmice) - Visualize incomplete and imputed data with the R package `ggmice`
```

Reply:
@kequach agree?

Reply:
Yes, that allows for further extensions to be added.